### DISCUSSION TITLE
hold the trailing zero

### DISCUSSION THREAD

**Discussion by lennylinatcnl** [davidgohel/flextable#618] [Q&A] [answered]
in `compose(j = 8, value = as_paragraph(minibar(value = B, max = sum(B), barcol = "blue"), B))` [as an example], The `B` column has values with the trailing zero.  as_paragraph() rounds my value to one decimal place.  I want to retain values with the trailing zeros.

**Comment by davidgohel** [comment, marked as answer]
Maybe 'fmt_double' or another 'fmt' function. Or 'as_chunk' instead of only B

**Reply by lennylinatcnl** [reply]
fmt_signif_after_zeros(x, digits = 3) works for me.  Thanks!

so it will look like:
`compose(j = 8,
         value = as_paragraph(fmt_signif_after_zeros(B, digits = 3), minibar(value = B max = sum(B), barcol = "blue")))`
